arm-datalake-analytics/job/2016-03-20-preview/swagger/job.json

```diff
@@ -0,0 +1,1017 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "DataLakeAnalyticsJobManagementClient",
+    "description": "Creates an Azure Data Lake Analytics job client.",
+    "version": "2016-03-20-preview",
+    "x-ms-code-generation-settings": {
+      "useDateTimeOffset": true,
+      "internalConstructors": true
+    }
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{accountName}.{adlaJobDnsSuffix}",
+    "parameters": [
+      {
+        "name": "accountName",
+        "description": "The Azure Data Lake Analytics account to execute job operations on.",
+        "required": true,
+        "type": "string",
+        "in": "path",
+        "x-ms-skip-url-encoding": true
+      },
+      {
+        "$ref": "#/parameters/adlaJobDnsSuffixInPath"
+      }
+    ]
+  },
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json",
+    "text/json",
+    "application/octet-stream"
+  ],
+  "produces": [
+    "application/json",
+    "text/json",
+    "application/octet-stream"
+  ],
+  "paths": {
+    "/Jobs/{jobIdentity}/GetStatistics": {
+      "post": {
+        "tags": [
+          "Job"
+        ],
+        "operationId": "Job_GetStatistics",
+        "description": "Gets statistics of the specified job.",
+        "parameters": [
+          {
+            "name": "jobIdentity",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "format": "uuid",
+            "description": "JobInfo ID."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
```

Comment:
@amarzavery it looks like there aren't any descriptions filled in for responses across the services right now. Is there guidance on what we should include in these descriptions (what the pattern should look like for the documentation inside of a response block)?